services/director/tests/unit/test_core_settings.py

```diff
@@ -54,7 +54,8 @@ def test_docker_container_env_sample(monkeypatch: pytest.MonkeyPatch):
         DIRECTOR_SELF_SIGNED_SSL_FILENAME=
         DIRECTOR_SELF_SIGNED_SSL_SECRET_ID=
         DIRECTOR_SELF_SIGNED_SSL_SECRET_NAME=
-        DIRECTOR_SERVICES_CUSTOM_CONSTRAINTS=node.labels.io.simcore.autoscaled-node!=true
+        DIRECTOR_SERVICES_CUSTOM_CONSTRAINTS=["node.labels.io.simcore.autoscaled-node!=true"]
```

Comment:
very minor: maybe call the env-var `...CUSTOM_PLACEMENT_CONSTRAINTS`, as then noone can confuse it with the docker custom resource constraints (vram etc)

Reply:
these are actually the same (VRAM and stuff). but otherwise yes I can change their name. will do actually.